pkgs/applications/audio/mellowplayer/default.nix

```
@@ -0,0 +1,65 @@
+{ cmake
+, fetchFromGitLab
+, lib
+, libnotify
+, pkgconfig
+, qt5
```

Please refer to the [Nixpkgs manual section](https://nixos.org/nixpkgs/manual/#sec-language-qt) for instructions about packaging Qt applications. The expression should not take `qt5` as an argument, but the individual Qt modules which are dependencies:

```suggestion
, qtbase
, qtdeclarative
, qtgraphicaleffects
, qtquickcontrols2
, qttools
, qtwebengine
```

---

I had it like this in the first version but decided to change it :facepalm: The reason I changed it is because I needed `mkDerivation` from `qt5`. I'll revert back to my first version then.